### ISSUE TITLE
Too long dbName

### ISSUE THREAD

**Issue by alexandrusavin** [Mockgoose/Mockgoose#60]
The `dbName` generated by mockgoose are too long and, in certain conditions, can trigger `namespace name generated from index name ... is too long (127 byte max)` 

https://github.com/Mockgoose/Mockgoose/blob/4de055ad2d791c5f4a309cb261f3e44f2769bcab/src/mockgoose.ts#L55

I'll try to create a PR to keep the same functionality while making the names shorter.

**Comment by alexandrusavin** [comment]
Yes, it is a mongodb limitation but it can be mitigated by not setting a uuidv4 value for the database name. Isn't this anyway a bit of overkill? A smaller random string would be more than enough to achieve the same end result. What is the probability of naming a database in production `mockgoose-temp-db-ugq234`?

**Comment by alexandrusavin** [comment]
Or am I missing the point of having a uuidv4 as dbname?

**Comment by alexandrusavin** [comment]
Don't really know. I would presume it's purpose is to avoid collisions if you'd run it alongside a production database. I would never recommend doing this on purpose but it did happen that by mistake I ran the tests with prod config.

I do agree that naming it `mockgoose-temp-db` is enough.

**Comment by sshtel** [comment]
I reproduced this issue too.

```
MongoError: namespace name generated from index name "mockgoose-temp-db-3410503c-e6c3-4a4e-a589-aa619167b652.reservenotices.$reservenotices_automatic_1_startPeriod_1_endPeriod_1_startTime_1_endTime_1_weekday_1" is too long (127 byte max)
```

I think removing prefix is OK since uuidV4 would be enough for local DB name without redundancy.

In my case, I don't know why `reservenotices.$reservenotices_automatic_1_startPeriod_1_endPeriod_1_startTime_1_endTime_1_weekday_1` is following after `mockgoose-temp-db-3410503c-e6c3-4a4e-a589-aa619167b652`
